sciserver_catalogs/environment.yml

```diff
@@ -37,8 +36,10 @@ dependencies:
   - cf_xarray
   - oceanspy==0.2.0
   - pip
+  - pooch 
   - pip:
     - git+https://github.com/MITgcm/xmitgcm.git
     - black-nb
```

Comment:
install using pip to get latest version with all our new code
git+https://github.com/hainegroup/oceanspy.git

Reply:
done